### ISSUE TITLE
Allow specifying custom URL parameters

### ISSUE THREAD

**Issue by zusorio** [DSD-DBS/capella-collab-manager#2041]
Xpra has a variety of URL parameters to change the behavior of the viewer. A user complained about the horizontal scrolling direction being reversed due to natural scrolling; this can be fixed by a URL parameter. We should allow users to specify custom URL parameters that are automatically used for every session.

**Comment by MoritzWeber0** [comment]
Makes sense, but we need to think about the UX. We should not allow users to define arbitrary URL parameters, which could also lead to a broken redirect URL. Instead, it should be limited to specific configuration options.

**Comment by zusorio** [comment]
Now that the horizontal scrolling issue is fixed, I don't see much priority for this anymore. Can you think of any reasons we should still pursue this, or can I close this @MoritzWeber0

**Comment by MoritzWeber0** [comment]
Can be closed
